agent/src/iconfucius/siwb.py

```
@@ -58,6 +58,94 @@
+            canister_id=ODIN_SIWB_CANISTER_ID,
+            candid_str=ODIN_SIWB_CANDID,
+        )
+        # Dummy query — will return Err but proves canister responds
+        siwb.siwb_get_delegation(
+            "bc1p0000000000000000000000000000000000000000000000000000",
+            bytes(32),
+            0,
+            verify_certificate=get_verify_certificates(),
+        )
+        result = True
+    except Exception as exc:
```

Ruff (0.15.4)

[warning] Do not catch blind exception: `Exception`

(BLE001)